src/Databases/DataLake/StorageCredentials.h

```diff
@@ -42,10 +41,16 @@ class S3Credentials final : public IStorageCredentials
         engine_args.push_back(std::make_shared<DB::ASTLiteral>(session_token));
     }
 
+    void setRegion(const std::string & region_)
```

Comment:
Why not just pass it through constructor?

Reply:
Just removed it, it was unused.